### PR TITLE
Remove myself from the build cop rotation 🧘❤️

### PR DESCRIPTION
**Pull request by bobcatfish**
# Changes

Lately I've been looking for ways to lighten my load. In stepping back
and looking for things I could stop doing, it feels reasonable to me
for me to stop build copping, at least for a while, so I'd like to
remove myself from the rotation.

I've made the change after next week so I can do one more rotation and
because this changes the entire schedule after that point so I want to
minimize the disruptions to everyone else.

Thanks! ❤️

This this changes the whole rotation after that point, heads up @afrittoli @wlynch @dibyom @nikhil-thomas @savitaashture @sbwsg @jerop @vdemeester 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: bots/buildbot/rotation.csv [hunk 1]
```diff
@@ -108,44 +11,45 @@ Date,User
 2020-10-21,wlynch
 2020-10-22,christiewilson
 2020-10-23,dibyo
+Date,User
```
Comment (sbwsg) [inline bots/buildbot/rotation.csv:14]: this is interesting - the Date,User heading appears twice in the file now.  must be a bug in the rotation generator :/

**Review by sbwsg — APPROVED** [review]

File: bots/buildbot/rotation.csv (hunk 1 above)
Comment (bobcatfish) [reply bots/buildbot/rotation.csv]: whoops, thanks for catching that!

**Review by vdemeester — COMMENTED** [review]
/lgtm
/meow

**Pull request merged**